src/main.js

```diff
@@ -4,6 +4,7 @@ let log = require('./log');
 let aws = require('aws-sdk');
 let _ = require('lodash');
 let path = require('path');
+let fs = require('fs');
```

Comment:
Let's use `mz/fs` instead.  It's a module that wraps the entire node standard library in promises.
```
let fs = require('mz/fs');`
```